src/ontology/edits/oeo-shared.omn

```diff
@@ -1078,7 +1078,9 @@ issue: https://github.com/OpenEnergyPlatform/ontology/issues/660
 pull request: https://github.com/OpenEnergyPlatform/ontology/pull/665
 move to oeo-shared
 issue: https://github.com/OpenEnergyPlatform/ontology/issues/1030
-pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1101",
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1101
+adjust definition
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1165",
```

Comment:
Please include also the issue in the term tracker

Reply:
See comment above.